### DISCUSSION TITLE
brew autoupdate showing "Error: Unknown command: autoupdate" on brand new install

### DISCUSSION THREAD

**Discussion by honeybyhomo** [Homebrew/discussions#5200] [Getting started] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.12
ORIGIN: https://github.com/Homebrew/brew
HEAD: 780fbbc65e90fbe09629aba180a1839e9e7dbaf2
Last commit: 2 days ago
Core tap JSON: 13 Mar 13:00 UTC
Core cask tap JSON: 13 Mar 13:00 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 3.1.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: octa-core 64-bit arm_ibiza
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 14.4-arm64
CLT: 15.3.0.0.1.1708646388
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Just installed homebrew on a brand new computer. Installed a lot via brewfile (bundle) – everything is working well. But `brew autoupdate` doesn't work and just says `Error: Unknown command: autoupdate`

**Comment by SMillerDev** [comment, marked as answer]
`brew autoupdate` is an external command and not part of brew. Unless you tap https://github.com/DomT4/homebrew-autoupdate in your Brewfile, it won't be available.

**Reply by honeybyhomo** [reply]
Ah, I thought that since it's `brew autoupdate` that it would be part of "standard brew". Thanks!
